frontend/src/pages/MilestonesEditor.module.scss

```diff
@@ -45,6 +45,18 @@
   width: 100%;
   border-top: 1px solid $COLOR_BLACK10;
   padding: 13px;
+  &.loading {
+    svg {
+      width: 21px;
+      height: 23px;
+      position: relative;
+      top: 1px;
+
+      path {
+        fill: $COLOR_BLACK40;
+      }
+    }
+  }
```

Comment:
This should be unnecessary, if the styles are applied on the svg button disabled styles